backend/routes_reports.py

```
@@ -355,3 +359,551 @@ def department_usage_report():
+                CycleCountResult.counted_at >= start_date,
+                CycleCountResult.has_discrepancy == True
+            ).join(CycleCountItem)
+
+            # Apply filters if provided
+            if discrepancy_type:
+                query = query.filter(CycleCountResult.discrepancy_type == discrepancy_type)
+
+            if location:
+                query = query.filter(CycleCountItem.expected_location.ilike(f'%{location}%'))
+
+            discrepancies = query.order_by(CycleCountResult.counted_at.desc()).all()
```

Ruff (0.11.9)

Local variable `discrepancies` is assigned to but never used

Remove assignment to unused variable `discrepancies`

(F841)